modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/template_selector/page.jsp

```diff
@@ -100,8 +100,10 @@ Group ddmTemplateGroup = GroupLocalServiceUtil.getGroup(ddmTemplateGroupId);
 					basePortletURL: '<%= basePortletURL %>',
 					classNameId: '<%= classNameId %>',
 					dialog: {
+						destroyOnHide: true,
 						width: 1024
 					},
+					eventName: 'saveTemplate',
```

Comment:
@antonio-ortega, wouldn't be better add the portlet's namespace (`<portlet:namespace />`) as a prefix of the event's name?

Reply:
I was following the same [pattern](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js#L1369-L1371) we have for the default event in util.js, but I think it could be a good practice.